syntax = "proto3";
package nexus.meta_api.proto;

import "nexus/models/proto/typed_document.proto";

service Documents {
  rpc get (TypedDocumentRequest) returns (nexus.models.proto.TypedDocument) {}
}

message TypedDocumentRequest {
  string index_alias = 1;
  uint64 document_id = 2;
  uint32 position = 3;
  string mode = 4;
}
